packages/cli/internal/pkg/aws/cwl/stream_logs.go

```diff
@@ -52,12 +52,17 @@ func (c Client) StreamLogs(ctx context.Context, logGroupName string, streams ...
 }
 
 func parseEventLogs(events []types.FilteredLogEvent, latestTimestamp *int64) []string {
-	logs := make([]string, len(events))
-	for i, event := range events {
+	logsByStream := make(map[string][]string)
```

Comment:
Could you please explain this change?

Reply:
The logs returned from Cloudwatch can be interleaved between different log streams as the API is a "best efforts" API that tries not to interleave the logs between log groups. This change forces the streams to be returned back together. 

From the description of the PR: 
> Currently, logs are shown directly as configured from Cloudwatch. This can end up causing logs to be interleaved with one another which can be confusing. This PR forces the logs to be grouped together by the log group.
> API Source: https://github.com/aws/aws-sdk-go-v2/blob/service/cloudwatchlogs/v1.8.0/service/cloudwatchlogs/api_op_FilterLogEvents.go#L68